demos/nwqn_data_pull/lithops.yaml

```diff
@@ -8,7 +8,8 @@ aws:
 aws_lambda:
     execution_role: arn:aws:iam::807615458658:role/lambdaLithopsExecutionRole
```

Comment:
No, anyone could use it. It's possible this one requires you to possess USGS AWS credentials, but I can't confirm that without violating policy.

Reply:
@thodson-usgs can you double check whether this is an internal-only demo? If so then there is likely a better place to put it than in the public-facing package documentation. I suppose we could add a qualifier indicating that USGS credentials are required to use the demo, at a minimum...

Reply:
The instructions indicate that you're supposed to create your own execution_role, but this one will work for USGS employees, at least. I'd prefer to remove the account_id rather than suggest this is internal-only.

Reply:
OK, thanks for clarifying. Removing the account_id sounds like a good approach.